manubot/cite/util.py

```diff
@@ -218,101 +218,106 @@ def is_valid_citation(
-    as_bytes = standard_id.encode()
+    assert '@' not in citekey
+    as_bytes = citekey.encode()
     blake_hash = hashlib.blake2b(as_bytes, digest_size=6)
     digest = blake_hash.digest()
-    short_id = base62.encodebytes(digest)
-    return short_id
+    short_citekey = base62.encodebytes(digest)
+    return short_citekey
 
 
-def citation_to_citeproc(citation, prune=True):
+def citekey_to_csl_item(citekey, prune=True):
     """
-    Return a dictionary with citation metadata
+    Generate a CSL Item (Python dictionary) for the input citekey.
     """
-    citation == standardize_citation(citation, warn_if_changed=True)
-    source, identifier = citation.split(':', 1)
+    citekey == standardize_citekey(citekey, warn_if_changed=True)
+    source, identifier = citekey.split(':', 1)
 
     if source in citeproc_retrievers:
         citeproc_retriever = import_function(citeproc_retrievers[source])
         csl_item = citeproc_retriever(identifier)
     else:
-        msg = f'Unsupported citation source {source} in {citation}'
+        msg = f'Unsupported citation source {source!r} in {citekey!r}'
         raise ValueError(msg)
 
     from manubot import __version__ as manubot_version
     from manubot.cite.citeproc import (
-        citeproc_passthrough,
+        csl_item_passthrough,
         append_to_csl_item_note,
     )
 
     note_text = f'This CSL JSON Item was automatically generated by Manubot v{manubot_version} using citation-by-identifier.'
     note_dict = {
-        'standard_id': citation,
+        'standard_id': citekey,
     }
     append_to_csl_item_note(csl_item, note_text, note_dict)
 
-    short_id = get_citation_short_id(citation)
-    csl_item = citeproc_passthrough(csl_item, set_id=short_id, prune=prune)
+    short_citekey = shorten_citekey(citekey)
+    csl_item = csl_item_passthrough(csl_item, set_id=short_citekey, prune=prune)
 
     return csl_item
 
 
-def infer_citation_prefix(citation):
+def infer_citekey_prefix(citekey):
     """
-    Passthrough citation if it has a valid citation prefix. Otherwise,
-    if the lowercase citation prefix is valid, convert the prefix to lowercase.
-    Otherwise, assume citation is raw and prepend "raw:".
+    Passthrough citekey if it has a valid citation key prefix. Otherwise,
+    if the lowercase citekey prefix is valid, convert the prefix to lowercase.
+    Otherwise, assume citekey is raw and prepend "raw:".
     """
     prefixes = [f'{x}:' for x in list(citeproc_retrievers) + ['raw']]
     for prefix in prefixes:
-        if citation.startswith(prefix):
-            return citation
-        if citation.lower().startswith(prefix):
-            return prefix + citation[len(prefix):]
-    return f'raw:{citation}'
+        if citekey.startswith(prefix):
+            return citekey
+        if citekey.lower().startswith(prefix):
+            return prefix + citekey[len(prefix):]
+    return f'raw:{citekey}'
 
 
 def csl_item_set_standard_id(csl_item):
```

Comment:
Keeping `standard_id` here instead of `standard_citekey`?  It looks intentional per the documentation below.